### DISCUSSION TITLE
Database state filtering

### DISCUSSION THREAD

**Discussion by Nness** [ravendb/ravendb#12207] [Ideas]
Currently, there is display on how many database is online and offline within a cluster. But there is no filter for that, only filter it has is for same node filter.

When database is not multi-tenant, the amount of database could be same as amount of clients, which will take a bit of time to find offline database, either not using or disabled. e.g. Office 365 tenant

**Comment by ayende** [comment]
Good idea: https://issues.hibernatingrhinos.com/issue/RavenDB-16740
